artifactory/commands/npm/installorci_test.go

```diff
@@ -1,15 +1,19 @@
 package npm
 
 import (
-	biutils "github.com/jfrog/build-info-go/utils"
 	"os"
 	"path/filepath"
 	"strings"
 	"testing"
 
+	biutils "github.com/jfrog/build-info-go/utils"
+	testsutils "github.com/jfrog/jfrog-client-go/utils/tests"
+
 	"github.com/stretchr/testify/assert"
 )
 
+const authToken = "YWRtaW46QVBCN1ZkZFMzN3NCakJiaHRGZThVb0JlZzFl"
```

Comment:
This isn't actually token, as authTokens are also supported against specific repositories/scopes.

Not sure what would be better name for this, but using `authToken` might be confusing.